deps/orchestrator/Dockerfile

```diff
@@ -0,0 +1,97 @@
+FROM rust:1.81 AS builder
+
+# Install system dependencies
+RUN apt update && apt install -y  \
+    git \
+    libgmp3-dev \
+    software-properties-common \
+    wget \
+    bash \
+    nodejs \
+    npm
+
+# Set the working directory
+WORKDIR /usr/src
+
+# Clone the specific branch of the repository
+RUN git clone --branch unignore_cargo_lock --depth 1 https://github.com/madara-alliance/madara-orchestrator.git
+
+WORKDIR /usr/src/madara-orchestrator
+
+# Install Python 3.9
+RUN wget https://www.python.org/ftp/python/3.9.16/Python-3.9.16.tgz \
+    && tar xzf Python-3.9.16.tgz \
+    && cd Python-3.9.16 \
+    && ./configure --enable-optimizations \
+    && make altinstall \
+    && cd .. \
+    && rm -rf Python-3.9.16 Python-3.9.16.tgz
+
+# Install pip and venv for Python 3.9
+RUN wget https://bootstrap.pypa.io/get-pip.py \
+    && python3.9 get-pip.py \
+    && rm get-pip.py \
+    && python3.9 -m pip install virtualenv
+
+# Check rust version (this also installs version from rust-toolchain file)
+RUN rustup show
+
+# #############################################################
+# TODO : remove this step after snos build is sorted
+# Build cairo lang
+RUN cargo fetch
+RUN python3.9 -m venv orchestrator_venv
+RUN . ./orchestrator_venv/bin/activate
+RUN pip install cairo-lang==0.13.2 "sympy<1.13.0"
+RUN mkdir -p build
+RUN git submodule update --init --recursive
+RUN cd cairo-lang
+RUN cd ..
+RUN cairo-compile cairo-lang/src/starkware/starknet/core/os/os.cairo --output build/os_latest.json --cairo_path cairo-lang/src
```

Comment:
Wasnt this handle by cairo-lang DockerFile? 
🤔

Reply:
It was indeed! However, Orchestrator needs the OS to run the SNOS jobs. Since the docker image that was provided in the repo contained that step, I just took it as a source of inspiration but this step should be removed from this for sure.

Current status:
Docker image from cairo-lang is not used
Docker image from orchestrator handles OS + Orchestrator